### PR TITLE
Feature-IOPS-2202-Remove-Identifier in Codesystem FGMRemovalReason & Spell Checker

### PR DESCRIPTION
**Pull request by KazeemHamzat**
Feature-IOPS-2202-Remove-Identifier in Codesystem FGMRemovalReason & Spell Checker

### CODE REVIEW

**Comment by marcusfearnett-nhsdigital** [comment]
Reviewed changes. Fine.

**Review by marcusfearnett-nhsdigital — DISMISSED** [review]
Approve.

File: codesystems/CodeSystem-England-FGMRemovalReason.xml [hunk 1]
```diff
@@ -21,8 +17,8 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="A CodeSystem used for the removal of FGM Flag" />
-  <purpose value="A codesystem used for the removal of FGM Flag" />
+  <description value="A CodeSystem used for the removal of FGM Flag." />
+  <purpose value="A codesystem used for the removal of FGM Flag." />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-FGMRemovalReason.xml]: codesystem -> CodeSystem

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

**Review by digitalhealthmartin — APPROVED** [review]

**Review by SheenaSharma — APPROVED** [review]

**Comment by KazeemHamzat** [comment]
Further work has been done via another PR: https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/406

**Pull request closed without merging**